Formula/geocouch.rb

```diff
@@ -1,11 +1,10 @@
 class Geocouch < Formula
```

Comment:
FYI, we don't have any migration  mechanism for this kind of change. Historically, we would just remove this formula.